mitodl/mitxonline
# Use site name variable (MITx Online) for page title

# Description:
The site name variable (default: MITx Online) should be used for all pages in the application.  Currently, the index page is hardcoded with "mitxonline".

# Screenshots:
![Screenshot 2023-09-14 at 7 48 37](https://github.com/mitodl/mitxonline/assets/8311573/faffdb62-fb44-463c-858b-6f38e7d66e55)


# How can this be tested?

1. Take this branch.
2. Load any pages in the application and ensure that title is "MITx Online"
